cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc

```diff
@@ -16,25 +16,25 @@ The list of addressed issues are not cumulative; only the issues that are fixed
 |Description
 
 |*PCSUP-9241*
-+++<draft-comment></draft-comment>+++
+// #39394
 |For the AWS US Gov region, alerts are not forwarded successfully to the AWS Security Hub integration.
-+++<draft-comment>With this fix, the correct AWS product ARN for US and China regions are used.</draft-comment>+++
+//With this fix, the correct AWS product ARN for US and China regions are used.
 
 
 |*PCSUP-11309*
-+++<draft-comment></draft-comment>+++added update 2
+//added Kepler Update 2
 |The userinput:[--tarball] option in twistcli does not scan for compliance checks. Currently, only vulnerabilities are detected successfully.
 
 
 |*—* 
 
-+++<draft-comment>25370</draft-comment>+++
+// #25370
 |For custom compliance checks for Kubernetes and OpenShift on CRI-O, if you have configured *Reported results* to show both passed and failed checks, in the event that a check doesn’t run, Prisma Cloud still reports it as passed.
```

Comment:
@Pubs-MV @divyabhushan 
This now works, see the last comment here https://redlock.atlassian.net/browse/CWP-25370?focusedCommentId=606551
We can remove it from the known issues

Reply:
Done. Removed it. Thanks